zap/src/main/java/org/zaproxy/zap/extension/brk/BreakpointsOptionsPanel.java

```diff
@@ -131,7 +197,11 @@ public void initParam(Object obj) {
         // Note param.alwaysOnTop will be null if the user hasn't specified a preference yet
         getCheckBoxAlwaysOnTop().setSelected(param.getAlwaysOnTop() != Boolean.FALSE);
         getCheckBoxInScopeOnly().setSelected(param.isInScopeOnly());
+        getCheckBoxShowIgnoreFilesButtons().setSelected(param.getIsToShowIgnoreFilesButtons());
         getButtonMode().setSelectedIndex(param.getButtonMode() - 1);
+        getJavascriptUrlRegexField().setText(param.getJavascriptUrlRegex());
+        getCssAndFontsUrlRegexField().setText(param.getCssAndFontsUrlRegex());
+        getMultimediaUrlRegexField().setText(param.getMultimediaUrlRegex());
```

Comment:
Should call `discardAllEdits()` after setting each value.

Reply:
ok, can you explain why?

Reply:
Memory (discard edits no longer needed) and usability (restore to clean state).